CareKit/SymptomTracker/OCKSymptomTrackerViewController.m

```diff
@@ -84,8 +84,7 @@ - (void)viewDidLoad {
                                                                               style:UIBarButtonItemStylePlain
                                                                              target:self
                                                                              action:@selector(showToday:)];
-    self.navigationItem.rightBarButtonItem.tintColor = self.progressRingTintColor;
-    
```

Comment:
I am unable to reproduce this issue.
In `OCKSample`, a red tint color is applied to the entire window.
This custom tint color can be seen in the Symptom Tracker and the Connect Detail View Controller.